### DISCUSSION TITLE
Virgo | Laniakea Super Cluster Plugin

### DISCUSSION THREAD

**Discussion by oDinZu** [Stellarium/stellarium#3565] [Ideas]
> Laniakea: Hawaiian for "open skies" or "immense heaven"

Is there a plugin or way to view the Laniakea with Stellarium? I conducted a light search and  wasn't able to discover anything related.

**Comment by alex-w** [comment]
No, it's not possible

**Comment by gzotti** [comment]
If you just have an image that highlights something, you can fit it in as SkyImage via scripting. If it's a larger thing that spans more of the sky than just a few square degrees, you could make a HiPS allsky image.
